Rubberduck.Inspections/Concrete/UnreachableCaseInspection/ParseTreeExpressionEvaluator.cs

```diff
@@ -0,0 +1,163 @@
+﻿using Rubberduck.Parsing.Grammar;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Rubberduck.Inspections.Concrete.UnreachableCaseInspection
+{
+    public interface IParseTreeExpressionEvaluator
+    {
+        IParseTreeValue Evaluate(IParseTreeValue LHS, IParseTreeValue RHS, string opSymbol);
+        IParseTreeValue Evaluate(IParseTreeValue LHS, string opSymbol);
+    }
+
+    public class ParseTreeExpressionEvaluator : IParseTreeExpressionEvaluator
+    {
+        private readonly IParseTreeValueFactory _valueFactory;
+
+        private static Dictionary<string, Func<double, double, double>> MathOpsBinary = new Dictionary<string, Func<double, double, double>>()
+        {
+            [MathSymbols.MULTIPLY] = delegate (double LHS, double RHS) { return LHS * RHS; },
+            [MathSymbols.DIVIDE] = delegate (double LHS, double RHS) { return LHS / RHS; },
```

Comment:
One more operator is missing, I think - the integer division, which is used with the symbol `\`.

Reply:
Added integer division